apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/attach-focused-launch.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import * as React from 'react';
+import React from 'react';
```

Comment:
`react` has to be in scope, otherwise TS complains when using JSX.

I assume this was done to avoid shipping React and use whatever is in the page. That should be handled by webpack